xml/hwbp_registers.xml

```diff
@@ -549,6 +552,37 @@
         </field>
     </register>
 
+    <register name="External Trigger" short="exttrigger" address="0x7a1">
+        This register is accessible as \RcsrTdataOne when \FcsrTdataOneType is 7.
+
+        This trigger fires when a selected external debug trigger input
+        asserts (e.g. when a pulse is received).  One of up to 16 external
```

Comment:
I'd use "signals" instead of "asserts," and remove the parenthetical.

Reply:
I updated this and specified that it fires if "any" selected input signals to be explicit that this is an OR function.